packages/extension-chakra-storefront/src/pages/product-list/index.jsx

```diff
@@ -109,8 +109,7 @@ const ProductList = (props) => {
     const {
         pages: {ProductList: productListConfig},
         maxCacheAge: MAX_CACHE_AGE,
-        staleWhileRevalidate: STALE_WHILE_REVALIDATE,
-        search: searchConfig
```

Comment:
I take it we weren't using this `searchConfig`?

Reply:
yes, we don't need this data in this page anymore